Workbooks/Azure Web Application Firewall/WAF monitoring/WAF Monitoring.workbook

```diff
@@ -44,7 +42,7 @@
               "value::all"
             ],
             "value": [
-              "/subscriptions/27cafca8-b9a4-4264-b399-45d0c9cca1ab"
+              ""
```

Comment:
instead of empty string, should remove the value field+value entirely in a bunch of these place